submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+A popular simple alternative for steam turbine placement is attaching two to each heat exchanger without connecting anything with pipes. This provides an excess steam consumption capacity that remains unused unless your plant has steam storage (discussed below).
+
+And so all the component counts are covered. If you want to learn more about the mathematics involved here, or if you want a calculator to do most of it for you, the [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) is the resource I'd recommend.
+
+#### What do we consider when placing the components?
+
+In terms of where to build the nuclear plant as a whole, it is quite beneficial to build near or on water, because the plant needs tons of it. Being close to uranium is optional because it can be easily transported via trains.
+
+Next is the question of arranging the reactors. We already know about harnessing the neighbor bonus and calculating it using SRE diagrams. Given that we want to connect as many neighbors as possible, it would make sense to just have a huge spaceless grid of reactors, but the disadvantage of that is there being no space to set up a system for automatically refueling any of the reactors located in the middle, unless you use [a mod for that](https://mods.factorio.com/mod/logistics-reactor-next). Thus, in an ideal setup, all reactors have neighbors on three sides and have the fourth side open for inserters to add/remove fuel cells. The 2xN setup is massively popular because it achieves exactly this, for every reactor except for the corner ones.
+
+Your other big concern is fluid flow, and the main constraint is the heat flow along heat pipes. The farther away the heat gets from the reactors, the lower the maximum temperature the pipes can have. Since reactor temperatures cannot exceed 1000°C, heat pipes beyond a certain distance become too cold to operate heat exchangers. As [the wiki shows](https://wiki.factorio.com/Tutorial:Nuclear_power#Heat_pipes) and [this post effectively visualizes](https://www.reddit.com/r/factorio/comments/ge4y6c/heat_pipe_throughput_and_a_bonus_note_on_parallel/), a single pipe can heat a row of at most 21 heat exchangers, but you can improve the range here by making the pipe thicker and by adding heat exchangers on both sides of it instead of one side. Another tricked used by the community here is placing unfueled nuclear reactors to act as 5x5 heat pipes because this allows travelling the same distance with fewer entities and connecting to more heat exchangers from a single entity. In any case, it is best to keep your heat pipes short by keeping heat exchangers near the reactors.
```

Comment:
```suggestion
Your other big concern is fluid flow, and the main constraint is the heat flow along heat pipes. The farther away the heat gets from the reactors, the lower the maximum temperature the pipes can have. Since reactor temperatures cannot exceed 1000°C, heat pipes beyond a certain distance become too cold to operate heat exchangers. As [the wiki shows](https://wiki.factorio.com/Tutorial:Nuclear_power#Heat_pipes) and [this post effectively visualizes](https://www.reddit.com/r/factorio/comments/ge4y6c/heat_pipe_throughput_and_a_bonus_note_on_parallel/), a single pipe can heat a row of at most 21 heat exchangers, but you can improve the range here by making the pipe thicker and by adding heat exchangers on both sides of it, instead of only on one side. Another tricked used by the community here is placing unfueled nuclear reactors to act as 5x5 heat pipes because this allows travelling the same distance with fewer entities and connecting to more heat exchangers from a single entity. In any case, it is best to keep your heat pipes short by keeping heat exchangers near the reactors.
```